src/test/resources/projects/it1/reference/pom.xml

```diff
@@ -203,6 +295,31 @@
 						</execution>
 					</executions>
 				</plugin>
+				<!--This plugin's configuration is used to store Eclipse m2e settings only. It has no influence on the Maven build itself.-->
+				<plugin>
+					<groupId>org.eclipse.m2e</groupId>
```

Comment:
There is a better way now to do this. Cf. https://www.eclipse.org/m2e/documentation/release-notes-17.html